indico/modules/events/editing/client/js/editing/page_layout/MenuBar.jsx

```diff
@@ -6,58 +6,78 @@
 // LICENSE file for more details.
 import managementURL from 'indico-url:event_management.settings';
 import displayURL from 'indico-url:events.display';
+import editableTypeListURL from 'indico-url:event_editing.editable_type_list';
 
 import React from 'react';
+import {Link} from 'react-router-dom';
 import PropTypes from 'prop-types';
-import {Header, Icon} from 'semantic-ui-react';
-import {Translate} from 'indico/react/i18n';
+import {Header, Icon, Menu} from 'semantic-ui-react';
+import {Param, Translate} from 'indico/react/i18n';
+import Palette from 'indico/utils/palette';
+import {EditableTypeTitles} from '../../models';
 
 import './MenuBar.module.scss';
 
-export default function MenuBar({eventId, menuItems}) {
+export default function MenuBar({eventId, menuItems, type}) {
+  const renderEditableListMenu = () => {
+    if (location.pathname === editableTypeListURL({confId: eventId, type})) {
+      return (
+        <Menu.Item active>
+          <Translate>Editable list</Translate>
+        </Menu.Item>
+      );
+    }
+    return (
+      <Menu.Item>
+        <Link to={editableTypeListURL({confId: eventId, type})}>
+          <Translate>Editable list</Translate>
+        </Link>
+      </Menu.Item>
+    );
+  };
+
   return (
     <div styleName="menu-bar">
       <Header as="h2" styleName="header">
-        <Translate>Paper Editing</Translate>
+        <Translate>
+          <Param name="editableType" value={EditableTypeTitles[type]} /> Editing
+        </Translate>
       </Header>
-      <ul styleName="list">
-        <li>
-          <span styleName="capitalized">
+      <Menu vertical>{renderEditableListMenu()}</Menu>
```

Comment:
considering how simple `renderEditableListMenu` is I'd move its content here

Reply:
I'd prefer it in a separate function to avoid writing `if` code inside jsx twice.

Reply:
maybe worth making it a separate component (still in this same file) then?